packages/generate-timeline-docs/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@jspsych-dev/generate-timeline-docs",
```

Comment:
I think we will have to create the @jspsych-dev namespace on npm if we use this. I'm OK with that - just need to remember to do it before merge.